pyinfra/modules/pip.py

```diff
@@ -51,7 +51,7 @@ def virtualenv(
 def packages(
     state, host,
     packages=None, present=True, latest=False,
-    requirements=None, virtualenv=None
+    requirements=None, virtualenv_root=None
```

Comment:
Yeah it's a bit messy, but I think it's worth it for the user-friendlyness and no breaking changes.